packages/medusa/src/api-v2/admin/fulfillments/validators.ts

```diff
@@ -3,5 +3,42 @@ import { createSelectParams } from "../../utils/validators"
 
 export const AdminFulfillmentParams = createSelectParams()
 
-export const AdminCancelFulfillment = z.object({})
 export type AdminCancelFulfillmentType = z.infer<typeof AdminCancelFulfillment>
+export const AdminCancelFulfillment = z.object({})
+
+export type AdminCreateFulfillmentType = z.infer<typeof AdminCreateFulfillment>
+export const AdminCreateFulfillment = z.object({
+  location_id: z.string(),
+  provider_id: z.string(),
+  delivery_address: z.object({
+    company: z.string().optional(),
+    first_name: z.string().optional(),
+    last_name: z.string().optional(),
+    address_1: z.string().optional(),
+    address_2: z.string().optional(),
+    city: z.string().optional(),
+    country_code: z.string().optional(),
+    province: z.string().optional(),
+    postal_code: z.string().optional(),
+    phone: z.string().optional(),
+    metadata: z.object({}).optional(),
+  }),
+  items: z.array(
+    z.object({
+      title: z.string(),
+      sku: z.string(),
```

Comment:
Are both an SKU and barcode required?

Reply:
According to the module, both are required. Any reason why it wouldn't be?

Reply:
@riqwan both are unique identifiers, so I was wondering if it's enough to have one or the other